docs/proposals/edgemesh-design.md

```diff
@@ -46,7 +46,7 @@ The cloud native and microservice architecture is becoming more and more popular
 * Application access service with the same mechanism no matter it located on cloud or edge
 
 ### Service discovery on edge
-* Use iptables to redirect inbound and outbound traffic to the EdgeHub/ServiceBus. Then do service discovery in EdgeHub/ServiceBus by pick up the right POD from the service definition. The idea is similar with what istio is doing in the init_container: https://github.com/istio/istio/blob/master/tools/packaging/common/istio-iptables.sh
+* Use iptables to redirect inbound and outbound traffic to the EdgeHub/ServiceBus. Then do service discovery in EdgeHub/ServiceBus by pick up the right POD from the service definition. The idea is similar with what istio is doing in the init_container: https://github.com/istio/istio/blob/032be5b1ecfd5b087f623d76dca2879f8cd130be/tools/packaging/common/istio-iptables.sh
```

Comment:
how do you find this link `https://github.com/istio/istio/blob/032be5b1ecfd5b087f623d76dca2879f8cd130be/tools/packaging/common/istio-iptables.sh`, is this what action find or recommend?

Reply:
No, istio-iptables.sh is removed in https://github.com/istio/istio/pull/19273, the link is the last work path.